src/special.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
What is the intended use for this file? Math functions? A more descriptive name would be good.